builtin/game/chatcommands.lua

```diff
@@ -290,7 +290,7 @@ core.register_chatcommand("auth_reload", {
 
 core.register_chatcommand("remove_player", {
 	params = "<name>",
-	description = "Remove player data",
+	description = "Remove all player data from a player",
```

Comment:
This sounds like making < name > lose access to data, not removing their data.

Reply:
This doesn't make sense. What is ambigious about “remove all player data”?

Reply:
It's the "from a player part".

Reply:
While you're editing this maybe make it call `minetest.get_auth_handler().delete_auth(toname)` too

Reply:
What Ezhh wrote.

Reply:
It's ok as it is but maybe add more information: 'Remove player data from database'?

Reply:
@sfan5: Wouldn't it make more sense to change this in the `core.remove_player()` function instead?

Reply:
@Wuzzy2 No, `remove_player` behaves exactly as the documentation says and that's intended.